### DISCUSSION TITLE
Group Table View by Tracked By?

### DISCUSSION THREAD

**Discussion by sherryumlah** [community/community#46808] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've created a parent issue with a Task List of all the dependency sub-task issues.  
![Screenshot 2023-02-08 at 7 37 29 PM](https://user-images.githubusercontent.com/11161280/217692903-4ec6fcbf-0053-4c95-b54d-8acc47e279c3.png)

I also created a table view in my Github project and set it to group by "Tracked By".

I expected the view to group issues by their parent issue, but it's just listing all issues under a "No Tracked by" group header.![Screenshot 2023-02-08 at 7 36 12 PM](https://user-images.githubusercontent.com/11161280/217692747-d41bda79-a4ae-4332-8f81-22d3abcbc355.png)

Is there a way to group issues like this using the Tracked By and Tracks fields?

**Comment by mtcA6** [comment]
this works if you're using the beta tasklist

**Comment by md-asikuzzaman** [comment]
Group Table View by Tracked By #111

**Reply by md-asikuzzaman** [reply]
hello

**Comment by md-asikuzzaman** [comment]
Group Table View by Tracked By #222

**Reply by md-asikuzzaman** [reply]
hello
